lib/browser/api/menu.js

```diff
@@ -206,6 +207,8 @@ function generateGroupId (items, pos) {
 function removeExtraSeparators (items) {
   // fold adjacent separators together
   let ret = items.filter((e, idx, arr) => {
+    // check if element is null then return false so we don't fold null
+    if (e == null) return false
```

Comment:
In this function I would assume that none of the `items` are `null`s and just didn't do this check.